logging/testing/util.go

```diff
@@ -27,39 +26,14 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-var (
-	loggers = make(map[string]*zap.SugaredLogger)
-	m       sync.Mutex
-)
-
 // TestLogger gets a logger to use in unit and end to end tests
 func TestLogger(t *testing.T) *zap.SugaredLogger {
-	m.Lock()
-	defer m.Unlock()
-
-	logger, ok := loggers[t.Name()]
-
-	if ok {
-		return logger
-	}
-
 	opts := zaptest.WrapOptions(
 		zap.AddCaller(),
 		zap.Development(),
 	)
 
-	logger = zaptest.NewLogger(t, opts).Sugar().Named(t.Name())
-	loggers[t.Name()] = logger
-
-	return logger
-}
-
-// ClearAll removes all the testing loggers.
-// `go test -count=X` executes runs in the same process, thus the map
-// persists between the runs, but the `t` will no longer be valid and will
-// cause a panic deep inside testing code.
-func ClearAll() {
-	loggers = make(map[string]*zap.SugaredLogger)
+	return zaptest.NewLogger(t, opts).Sugar().Named(t.Name())
```

Comment:
Do we need named anymore?

Reply:
Yeah we don't need it anymore, since we are not caching the loggers by name.